.github/CODEOWNERS

```diff
@@ -1,9 +1,12 @@
 # Mondo code owners (Guardians of the main branch)
 
 # Removed @cmungall to stop him being added to all PRs
-*       @nicolevasilevsky 
+*       @sabrinatoro @twhetzel
 *.sparql @matentzn
 *.py	  @joeflack4
+*.md    @matentzn @twhetzel
```

Comment:
If this is documentation (including "curation" related documentation), I think I should be included here.

Reply:
Agreed and this has been updated now.